src/pages/proposals/ProposalCreate.vue

```diff
@@ -294,12 +294,7 @@ export default {
 
     refer (obj) {
       this.reference = obj
-      if (this.selectedConfig.type === 'Assignment') {
-        this.$store.commit('proposals/setRole', this.reference)
-        this.$store.commit('proposals/setAnnualUsdSalary', this.reference.salary ? this.reference.salary : this.reference.details_annualUsdSalary_a)
-        this.$store.commit('proposals/setMinDeferred', this.reference.minDeferred ? this.reference.minDeferred : this.reference.details_minDeferredX100_i)
-        this.$store.commit('proposals/setMinCommitment', this.reference.minCommitment ? this.reference.minCommitment : this.reference.details_minTimeShareX100_i)
-      } else if (this.selectedConfig.type === 'Assignment Badge') {
+      if (this.selectedConfig.type === 'Assignment Badge') {
```

Comment:
@arsenijesavic I see that you are using `Assignment Badge` string as type both here and in config file. It should be the same constant, instead of using a string. Plain typo here would cause difficult to detect bug